extensions/blocks/map/edit.js

```diff
@@ -185,6 +212,51 @@ class MapEdit extends Component {
 						] }
 					/>
 					<PanelBody title={ __( 'Map Settings', 'jetpack' ) }>
+						<BaseControl
+							label={ __( 'Height in pixels' ) }
+							id={ `block-jetpack-map-height-input-${ instanceId }` }
+						>
+							<input
+								type="number"
+								id={ `block-jetpack-map-height-input-${ instanceId }` }
+								onChange={ event => {
```

Comment:
These functions are little lengthy within JSX, I'd consider moving the out and referencing them. 🙂

Reply:
👍🏻 91e5b20bd